src/nautilus-view.h

```diff
@@ -408,4 +413,12 @@ void		  nautilus_view_action_show_hidden_files   (NautilusView      *view,
 
 GActionGroup *    nautilus_view_get_action_group           (NautilusView      *view);
 
+/* Functions to manage the scripts */
+void              run_script                               (GSimpleAction    *action,
+							    GVariant        *state,
+							    gpointer        user_data);
+ScriptLaunchParameters * script_launch_parameters_new      (NautilusFile       *file,
```

Comment:
If you follow my suggestion below, you most likely will not need to make these public.